### PR TITLE
Landline validation feature flagged sms

### PR DESCRIPTION
**Pull request by rparke**
Awaiting https://github.com/alphagov/notifications-api/pull/4136 approval and merge prior to review.

(will also do a squash/rebase once this is merged)

### CODE REVIEW

**Review by leohemsted — COMMENTED** [review]
looks good, just a couple of test suggestions really

File: tests/app/notifications/test_process_notification.py [hunk 1]
```diff
@@ -385,7 +386,77 @@ def test_persist_notification_with_international_info_stores_correct_info(
     assert persisted_notification.rate_multiplier == expected_units
 
 
+@pytest.mark.parametrize(
+    "recipient, expected_recipient_normalised, expected_prefix, expected_units",
+    [
+        ("020 7709 1001", "442077091001", "44", 1),  # UK
+        ("+442077091002", "442077091002", "44", 1),  # UK
+        ("020 7709 1000", "442077091000", "44", 1),  # UK
```
[the remark this answers is not in the document]
Comment (rparke) [reply tests/app/notifications/test_process_notification.py:394]: The first was meant to have spaces removed

File: tests/app/notifications/test_process_notification.py [hunk 2]
```diff
@@ -385,7 +386,77 @@ def test_persist_notification_with_international_info_stores_correct_info(
...
+
+@pytest.mark.parametrize(
+    "recipient, expected_recipient_normalised, expected_prefix, expected_units",
+    [
+        ("020 7709 1001", "442077091001", "44", 1),  # UK
+        ("+442077091002", "442077091002", "44", 1),  # UK
+        ("020 7709 1000", "442077091000", "44", 1),  # UK
+    ],
+)
+def test_persist_notification_without_send_to_landline_raises_invalidphoneerror(
```
[the remark this answers is not in the document]
Comment (rparke) [reply tests/app/notifications/test_process_notification.py:430]: added

**Review by leohemsted — APPROVED** [review]
looks good i think 👍

**Pull request merged**